File: eBarberShop.Models/Novost.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Text;

namespace eBarberShop.Models
{
    public class Novost
    {
        [Browsable(false)]
        public int NovostID { get; set; }
        [DisplayName("Dodao korisnik")]
        public string UposlenikIme => $"{Korisnik?.Ime} {Korisnik?.Prezime}";
        public string Naslov { get; set; }
        public string Sadrzaj { get; set; }
        public byte[] Thumbnail { get; set; }
        public DateTime DatumKreiranja { get; set; }
        [Browsable(false)]
        public int KorisnikID { get; set; }
        [Browsable(false)]
        public Korisnik Korisnik { get; set; }
    }
}

File: ebarbershop_mobile/lib/models/novost.dart
import 'package:json_annotation/json_annotation.dart';

import 'korisnik.dart';

part 'novost.g.dart';

@JsonSerializable()
class Novost{
  int? novostID;
  String? naslov;
  String? sadrzaj;
  String? thumbnail;
  String? opis;
  DateTime? datumKreiranja;
  int? korisnikID;
  Korisnik? korisnik;

  Novost();

    factory Novost.fromJson(Map<String, dynamic> json) => _$NovostFromJson(json);

    Map<String, dynamic> toJson() => _$NovostToJson(this);
}